WebApiCoreSeed.Infrastructure/RestClient/RestClient.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WebApiCoreSeed.Infrastructure.RestClient
+{
+    /// <summary>
+    /// <see cref="IRestClient"/> implementation relying on <see cref="HttpClient"/> and <see cref="Newtonsoft"/> to easily perform REST request and parse JSON responses

```

Comment:
Minor: should the second reference be "JsonConvert" (with the full qualifying name) instead of "Newtonsoft"?